src/cli/backup/backup.go

```diff
@@ -290,7 +290,16 @@ func genericDeleteCommand(
 
 	ctx := clues.Add(cmd.Context(), "delete_backup_id", bID)
 
-	r, _, _, _, err := utils.GetAccountAndConnect(ctx, pst, repo.S3Overrides(cmd))
+	provider, overrides, err := repo.GetStorageProviderAndOverrides(ctx, cmd)
+	if err != nil {
+		return Only(ctx, err)
+	}
+
+	r, _, _, _, err := utils.GetAccountAndConnect(
+		ctx,
+		pst,
+		provider,
+		overrides)
```

Comment:
Can't move `utils.GetStorageProviderAndOverrides` into `utils.GetAccountAndConnect` unfortunately. 

Main reason is that it'll break longevity tool which also uses `utils.GetAccountAndConnect`. This tool doesn't utilize corso config constructs like flags, config file. So reading from config file & applying flag overrides are not applicable here.

I'd prefer to leave this as is and address it with a proper config refactor later.

Reply:
Well, one more pitch before we leave it.  What if you created a new func to combine the two, left `utils.GetAccountAndConnect` so that longevity can call it.  Then have everything else call the new func?  It's a minor touch to the code, and hits all the places you're already updating, making it fairly within scope of this change.

Reply:
Thanks! That's a good idea. Although it'll not completely eliminate calling `utils.GetStorageProviderAndOverrides` from cli. We will need it for `GetConfigFromRepo` which is directly used by `init` & `connect` commands.

I'll see what works best. Should be quick.